pallets/offchain-worker/src/lib.rs

```diff
@@ -117,6 +117,14 @@ pub mod pallet {
 		type Reward: OnUnbalanced<PositiveImbalanceOf<Self>>;
 		type OcwQueryReward: Get<<<Self as Config>::Currency as Currency<<Self as frame_system::Config>::AccountId>>::Balance>;
 		type WeightInfo: weights::WeightInfo;
+
+		/// The maximum weight indicate the maximum weight used for data aggregation at the end of a session
+		type MaximumWeightForDataAggregation: Get<Weight>;
```

Comment:
These are to cap the weight for benchmarks right?

Reply:
may use it and the weight from benchmark to check if the MaximumCommitsPerSession is reasonable.